main.go

```diff
@@ -113,11 +113,12 @@ func NewGame(g *Game) {
 	g.Sprites[spritePlayer] = loadSprite("Player")
 	g.Sprites[spriteZombie] = loadSprite("Zombie")
 
+	// Load entities from map
 	entities := level.LayerByIdentifier("Entities")
-	playerLayer := entities.EntityByIdentifier("Player")
-	playerPosition := playerLayer.Position
```

Comment:
ok cool, I was gonna give this as review feedback but you were faster :ok_hand: nice